recipes/recipe/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "openmolcas" %}
+{% set version = "21.10" %}
+
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/Molcas/OpenMolcas/-/archive/v{{ version }}/OpenMolcas-v{{ version }}.tar.gz 
+  sha256: 46f30c47a9f30e030cf1a4ac8b73964da81657ea9093654da057a21c7fac63f3
+
+build:
+  number: {{ build }}
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - git
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake >=2.8.12
+    - make
+  host:
+    - python
+    - {{ mpi }}
+    - openblas-ilp64
+    - liblapack
+    - hdf5
+    - pyparsing
+    - gsl
+    - boost >=1.56
+  run:
+    - python
+    - {{ mpi }}
+    - libblas * *openblas
+    - openblas-ilp64
+    - liblapack
+    - hdf5
+    - gsl
+    - pyparsing
+
+test:
+  commands:
+    - pyopenmolcas --help
```

Comment:
if this is provided by python, you probably need to reproduce it under `#/build/entry_points`

Reply:
It's a weird console script that is generated by cmake